src/graphics/Stats.cpp

```diff
@@ -54,7 +54,7 @@ namespace Graphics {
 			frame.m_stats[idx] = statsFrame.at(name);
 		}
 
-		m_currentFrame = ++m_currentFrame % MAX_FRAMES_STORE;
+		m_currentFrame = (m_currentFrame + 1) % MAX_FRAMES_STORE;
```

Comment:
I did not expect this expression to trigger a warning, but after thinking about it I concluded the compiler is right: it does not make sense to modify the value (the increment operator) just to modify it after the modulo operation. Using a simple sum makes to code more readable and less bug prone in case of refactor, with [the same machine code](https://godbolt.org/z/rUDbEc).